Sources/IO/Geometry/GLTFImporter/index.js

```diff
@@ -1,25 +1,147 @@
 import macro from '@kitware/vtk.js/macros';
+import DataAccessHelper from 'vtk.js/Sources/IO/Core/DataAccessHelper';
+import Constants from 'vtk.js/Sources/IO/Geometry/GLTFImporter/Constants';
+
+// Enable data soure for DataAccessHelper
+import 'vtk.js/Sources/IO/Core/DataAccessHelper/LiteHttpDataAccessHelper'; // Just need HTTP
+
+// ----------------------------------------------------------------------------
+// vtkGLTFImporter methods
+// ----------------------------------------------------------------------------
 
 function vtkGLTFImporter(publicAPI, model) {
   //
   model.classHierarchy.push('vtkGLTFImporter');
+
+  if (!model.dataAccessHelper) {
+    model.dataAccessHelper = DataAccessHelper.get('http');
+  }
+
+  // To support destructuring
+  if (!model.compression) {
+    model.compression = null;
```

Comment:
should be done within `DEFAULT_VALUES` (or `function defaultValue`)

Reply:
should be fixed